src/createfeedback/Create-Feedback.scss

```diff
@@ -20,10 +20,29 @@
     display: flex;
     flex-direction: column;
 
+    h2 {
+      color: $darker-gray;
+      font-size: 13px;
```

Comment:
It's best practice to have these in `rems` and not pixels because of font scaling. See if you can convert them. You can either use a scss function or manually convert them (base is 16) => This example would be 13 / 16 = 0.8125rem;